docs/pages/desktop-access/getting-started.mdx

```diff
@@ -27,104 +25,128 @@ This guide requires you to have:
+Next, in the same PowerShell prompt, enter the following commands to limit your service account's permissions.
 
-<Admonition
-  type="note"
-  title="Default User Permissions"
->
-  By default, domain users have Read and List permissions throughout Active Directory. If you have configured
-  any restrictions to Domain Users, you may have to adjust which groups this account is a member of.
-</Admonition>
+```powershell
+# Save your domain's distinguished name to a variable.
+$DomainDN=$((Get-ADDomain).DistinguishedName)
 
-<Figure align="left" bordered caption="Create Service Account">
-  ![Create Service Account](../../img/desktop-access/ad-new-user.png)
-</Figure>
+# Create the CDP/Teleport container.
+# If the command fails with "New-ADObject : An attempt was made to add an object to the directory with a name that is already in use",
+# it means the object already exists and you can move on to the next step.
+New-ADObject -Name "Teleport" -Type "container" -Path "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN"
+
+# Gives Teleport the ability to create LDAP containers in the CDP container.
+dsacls "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CC;container;"
+# Gives Teleport the ability to create and delete cRLDistributionPoint objects in the CDP/Teleport container.
+dsacls "CN=Teleport,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;cRLDistributionPoint;"
+# Gives Teleport the ability to write the certificateRevocationList property in the CDP/Teleport container.
+dsacls "CN=Teleport,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN " /I:T /G "$($SamAccountName):WP;certificateRevocationList;"
+# Gives Teleport the ability to create and delete certificationAuthority objects in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;certificationAuthority;"
+# Gives Teleport the ability to write the cACertificate property in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):WP;cACertificate;"
+
+```
+
+## Step 2/6. Prevent the service account from performing interactive logins
 
-Now right click the same container you did before, and select `New > Group`. Choose a Group name like `svc-teleport-group` and ensure that “Global”
-is selected for “Group scope”, and “Security” for “Group type”, then hit "OK". Make note of the Group name you chose, as it will
-be used later for the PowerShell variable `$TeleportLDAPGroupName`.
+<Admonition type="note" title="gpupdate.exe">
+  Throughout this step and the next one, you will be modifying GPO's, and
+  sometimes GPO modifications can take some time to take effect. You can force
+  your GPO changes to take effect immediately at any time by opening a
```

Comment:
Note that this only forces the GPO changes to synchronize on the current host, not across the entire fleet. Is there a better way to word this?